### PR TITLE
Fixes inconsistent parentheses in power

### PR DESCRIPTION
**Pull request by Ryand1234**
#### References to other Issues or PRs
Fixes #6975

#### Release Notes

 
* printing 
   * fixed the parenthesization of powers in the latex printer

### CODE REVIEW

**Comment by Ryand1234** [comment]
Can someone tell me why test is failing?

**Comment by asmeurer** [comment]
There is some issue in the build for the pdf docs, which uses LaTeX. I wonder if `latex` is used somewhere to generate part of the documentation.

File: sympy/printing/latex.py [hunk 1]
```diff
@@ -616,22 +616,31 @@ def _print_Pow(self, expr):
...
                 return self._helper_print_standard_power(expr, tex)
 
     def _helper_print_standard_power(self, expr, template):
         exp = self._print(expr.exp)
         # issue #12886: add parentheses around superscripts raised
         # to powers
         base = self.parenthesize(expr.base, PRECEDENCE['Pow'])
-        if '^' in base and expr.base.is_Symbol:
-            base = r"\left(%s\right)" % base
+        if '^' in base :
```
Comment (asmeurer) [inline sympy/printing/latex.py]: Checking for substrings doesn't seem very robust. What does the logic for pprint look like?

Comment (Ryand1234) [reply sympy/printing/latex.py]: What i understand is that first the expression is parenthesized if necessary then the side brackets **'|'** are added to the picture of expression. And then variables are filled in those brackets.

Comment (Ryand1234) [reply sympy/printing/latex.py]: Actually i manipulated the previous written logic to get the parentheses around necessary expression.

**Comment by Ryand1234** [comment]
Ping @asmeurer

**Comment by Sc0rpi0n101** [comment]
I don't think this fix needs to be included in the release notes. 

Also, can you squash your commits into one with a little description of what was actually changed.

**Comment by asmeurer** [comment]
It can be included in the release notes, but make it relevant to the change for the end-user (like "fixed the parenthesization of powers in the latex printer").

**Comment by oscarbenjamin** [comment]
This has merge conflicts now.

File: sympy/printing/latex.py [hunk 2]
```diff
@@ -616,22 +616,35 @@ def _print_Pow(self, expr):
...
+            exp = r"{%s}" % exp
+        elif expr.base.is_Pow:
+            if isinstance(expr.exp,Add) or expr.exp.is_Integer or expr.exp.is_Rational :
+                exp = r"{%s}" % exp
+            else:
+                exp = r"\left({%s}\right)" % exp
+        else:
+            exp = r"{%s}" % exp
+
+        ''' checks if base expression contains '^' and it is not considered
```
Comment (asmeurer) [inline sympy/printing/latex.py:642]: Use # for comments.

**Comment by czgdp1807** [comment]
@Ryand1234 Are you still working on this PR? Please resolve the conflicts.

**Comment by Ryand1234** [comment]
I'll create a new PR with new branch because there is some problem in this branch.

**Pull request closed without merging**